docs/swarm/admin_guide.md

```diff
@@ -32,7 +32,7 @@ This article covers the following swarm administration tasks:
 * [Forcefully removing a node](#force-remove-a-node)
 * [Recovering from disaster](#recover-from-disaster)
 
-Refer to [How swarm mode nodes work](how-swarm-mode-works/nodes.md)
+Refer to [Hows nodes work](how-swarm-mode-works/nodes.md)
```

Comment:
@rogaha  PTAL
Sorry, and I have removed the 's' from this commit and updated this PR.
Thanks!